MR docs update (#4544)
iterative/dvc.org: content/docs/studio/user-guide/model-registry/add-a-model.md

You can add models from any ML project to the model registry. To add a model to
your model registry, Iterative Studio creates an annotation for it in a

> `dvc.yaml` file in your Git repository. You can also use [dvclive] to log your
> model during the training process or edit `dvc.yaml` directly. To add models
> using Iterative Studio, watch this tutorial video or read on below:

Reviewer: It would be nice to explain all of these options in more detail, but one quick fix would be to call them out here more clearly like this:

```suggestion
`dvc.yaml` file in your Git repository. You can add a model in any of the following ways:

1. Log your model during the training process using [dvclive].
2. Edit `dvc.yaml` directly.
3. Use the Iterative Studio interface (watch this tutorial video or read on below).
```

You can add models from any ML project to the model registry. To add a model to
your model registry, Iterative Studio creates an annotation for it in a
`dvc.yaml` file in your Git repository. You can add a model in any of the
following ways:

1. Log your model during the training process using [dvclive].

> 2. Edit `dvc.yaml` directly.

Reviewer: ```suggestion
2. Edit [`dvc.yaml`](/doc/user-guide/project-structure/dvcyaml-files#artifacts) directly.
```

   </admon>

3. Enter the path to `dvc.yaml` the model will be added to. Adding your model to
   non-root `dvc.yaml` can be helpful if you develop this ML model in a specific
   subfolder or if this repo is a

>    [monorepo](/doc/studio/user-guide/projects-and-experiments/configure-a-project#monorepo).

Reviewer: Is this handled already at the project level? Or are we adding an additional option to pick a subfolder when you add a model?

Author: Now in "add a model" user select from views => he's selecting repo + subdir

Reviewer: Is there anywhere with context about this decision? Why did we decouple it from the project?

     should be saved.

5. Provide labels for your model. For example, if your model is about reviewing
   sentiment analysis using natural language processing, one of the labels may
   be `nlp`. You can provide multiple labels as a comma separated list. Eg,
   `nlp, sentiment_analysis`.

6. Optionally, add a brief description for your model.

7. Enter a Git commit message. Then, select the branch to commit to. You can
   commit to either the base branch or a new branch. Iterative Studio will
   commit the changes to the selected branch. If you commit to a new branch,
   Iterative Studio will also create a Git pull request from the new branch to
   the base branch.

8. Now, click on `Commit changes`.

9. At this point, the new model appears in the models dashboard.

10. In your Git repository, you will find that an entry for the new model has
    been created in the `dvc.yaml` file in the repository's root. If you had
    committed to a new branch, a new pull request (or merge request in the case
    of GitLab) will also have been created to merge the new branch into the base
    branch.
11. If you had added a model from a cloud storage, the following will also

>     happen before the commit is created:

Reviewer: Minor: I don't think these are actual steps, and it makes this list longer than it needs to be. Can we take out the numbering and leave these in as an explanation of what you see after you complete these steps?

Reviewer: Thanks for changing this @aguschin! Again, this is minor and not a blocker, but you could do the same for the register and assign pages (they include steps that are really about explaining what you see afterwards).

    - If the specified directory does not exist yet, it will be created.
    - Iterative Studio will import the model to the repository by executing
      `dvc import-url <remote_path> <directory_path>/<filename from remote_path> --no-exec`.

> [connected repository]:

Reviewer: ```suggestion
[connected project]:
```
